examples/url/main.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2014 Hajime Hoshi
```

Comment:
Not sure if this is needed. Plus, it depends on external resources...

Reply:
The changes are cosmetic differences from original files.

Reply:
These are not changes, these are additions. And this is a code duplication which doesn't act as a useful example, in my opinion.
I'll let Hajime decide if this is needed or not.

Reply:
I agree, "url" Example is not needed (I thought you were talking of "Copyright 2014 Hajime Hoshi").